app/src/main/java/org/simple/clinic/teleconsultlog/teleconsultrecord/TeleconsultRecordPrescribedDrug.kt

```diff
@@ -0,0 +1,30 @@
+package org.simple.clinic.teleconsultlog.teleconsultrecord
+
+import androidx.room.Dao
+import androidx.room.Entity
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy.REPLACE
+import androidx.room.PrimaryKey
+import androidx.room.Query
+import java.util.UUID
+
+@Entity
+data class TeleconsultRecordPrescribedDrug(
+    @PrimaryKey
+    val teleconsultRecordId: UUID,
```

Comment:
This should also be a foreign key with a delete cascade action into the `TeleconsultRecord` table since we cannot have this without creating a record first.